modelforge/tests/test_ani.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 from modelforge.tests.helper_functions import setup_potential_for_test
 
+file_path = "modelforge/tests/data/torchani_parameters.state"
```

Comment:
can you change this to use "from importlib import resources" to get the absolute path.